shared.utils.ts

```diff
@@ -0,0 +1,45 @@
+import { readFileSync } from 'node:fs';
+
+/**
+ * Read a JSON file with canister IDs and transform it into a dictionary. Optionally add prefix to all dictionary keys.
+ */
+export const readCanisterIdsFromJSONFile = ({
```

Comment:
nth: can be shortened `readCanisterIds`. From IC dev perspective, it's the standard to read ids from a file

Reply:
Agreed.